spec.bs

```diff
@@ -334,9 +372,12 @@ to no longer be entitled to the purchase after it is consumed.
     be formatted as shown in the [[#querying-item-details]] example above for
     display to the user. It is expected to have been localized for the user by
     the {{Window/getDigitalGoodsService(serviceProvider)/serviceProvider}}.
+  * {{ItemDetails/type}} is either |"product"| or |"subscription"|.
```

Comment:
Maybe preferable to say it's one of the values in the "ItemType" enum? In case we add more.